internal/globalMerge.ecl

```diff
@@ -155,13 +155,13 @@ EXPORT globalMerge := MODULE
               id = id - lastgroupend;
               pid = pid - lastgroupend;
               *(uint32_t *)pos = wi;
-              pos += sizeof(uint32_t);
+              pos =(uint32_t *)pos + sizeof(uint32_t);
```

Comment:
It works, but I think it is a bit over-engineered and it would be much clearer if you change the line 141 from this:
   void * pos;
to this:
   byte * pos; 
only and it will automatically fix all other problem where 'pos += sizeof(uint32_t);' appears.

Moreover 'byte * pos;' would be correct because it gets its initial value with 'row' and the 'row' definition in line 134 is 'byte *'.

 (I think the 'unint8_t *' instead of 'byte *' would be clearer and future proof.)

Reply:
I changed the type of pos to unit8_t. Please test if necessary.